drivers/serial/uart_native_posix.c

```diff
@@ -398,7 +403,7 @@ static void np_add_uart_options(void)
 		"Automatically attach to the UART terminal"},
 		{false, false, false,
 		"attach_uart_cmd", "\"cmd\"", 's',
-		(void *)&auto_attach_cmd, NULL,
+		(void *)&auto_attach_cmd, auto_attach_cmd_cb,
 		"Command used to automatically attach to the terminal, by "
```

Comment:
Would you be so kind to add a "(Implies auto_attach)" to the this help description, so it is clearer for users.